test/alternator/test_export.py

```
@@ -0,0 +1,176 @@
+def test_export_table_unsupported_incremental(test_table_s_for_export_only, scylla_only):
+    client = test_table_s_for_export_only.meta.client
+
+    with pytest.raises(ClientError, match='ValidationException.*[eE]xportType'):
+        client.update_continuous_backups(
+            TableName=test_table_s_for_export_only.name,
+            PointInTimeRecoverySpecification={'PointInTimeRecoveryEnabled': True}
+        )
+
+
+# Test that incremental export is rejected.
+def test_export_table_unsupported_incremental(dynamodb, test_table_s_for_export_only, scylla_only):
```

Ruff (0.16.3)

[warning] Unused function argument: `dynamodb`

(ARG001)